modules/core/include/opencv2/core/hal/intrin_cpp.hpp

```diff
@@ -2080,6 +2080,14 @@ template<int n> inline v_reg<float, n> v_cvt_f32(const v_reg<int, n>& a)
     return c;
 }
 
+template<int n> inline v_reg<float, n> v_cvt_f32(const v_reg<uint, n>& a)
+{
+    v_reg<float, n> c;
+    for (int i = 0; i < n; i++)
+        c.s[i] = (float)a.s[i];
```

Comment:
please use c++ stile cast  (as it's easier to search for than c style casts)

Reply:
let left this "as is" (this file prefers C style casts)

Reply:
ok

Reply:
This code was removed.